src/hotspot/cpu/aarch64/stubGenerator_aarch64.cpp

```diff
@@ -32,12 +32,14 @@
 #include "gc/shared/tlab_globals.hpp"
 #include "interpreter/interpreter.hpp"
 #include "memory/universe.hpp"
+#include "atomic_aarch64.hpp"
```

Comment:
I think the conventions is to put the includes in alphabetic order after `#include "precompiled.hpp"`

Reply:
Good catch, thanks